packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -348,6 +361,7 @@ interface RuleListener {
   TSProtectedKeyword?: RuleFunction<TSESTree.TSProtectedKeyword>;
   TSPublicKeyword?: RuleFunction<TSESTree.TSPublicKeyword>;
   TSQualifiedName?: RuleFunction<TSESTree.TSQualifiedName>;
+  // TSQuestionToken?: RuleFunction<TSESTree.TSQuestionToken>;
```

Comment:
this should be handled by `Token`, shouldn't it?

Reply:
No idea, just saw it in `AST_NODE_TYPES`.

Reply:
ahh yes, it shouldn't be in there.
Feel free to remove it from `AST_NODE_TYPES`.